_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,172 @@
+
+### Manage Secrets
+
+In software development, certain parts of the final code may be a corporate secret, such as a private key to a computer resource, passwords, etc.
+
+These must never be included in the public source code itself but rather be managed separately.
+
+Since, software source code is nothing more than structured text that can be understood by a computer, the same concept of pulling out secrets parts may be taken with other types of documents.
+
+This means that when a small part of your work must not be shared publicly, you can exclude it and store it in a secure location instead of locking down the entire document.
+
+This way, the material that is unclassified can still be made available publicly and may be collectively worked on while the parts that are to be secured can be edited separately only by the select individuals who are authorized.
+
+### Manage Leaks
+
+In software development, we expect that there will be a security issue at some point, while we continue to strive to minimize that eventuality.
+
+By putting in place a remediation process, we ensure that even if there was to be a leak, appropriate mesures are in place to address it as quickly as possible.
+And we make sure that everyone knows how to run through the process, if it's not fully automated.
+
+In the optic of traditional text documents and presentations, reducing the amount of specific data used may help minimize the risks of leaks.
+However, someone could inadvertently type a password or personal information in a document.
+
+If that was the case, since it is not software related and automation may not be as easily possible, it would be important to have a clearly established protocol for your team members to address the data leak.
+
+### Open up at Inception
+
+Part of working in the open is accepting that we don't know everything and that others may be able to jump in and help at any time.
+By making your documents open at their inception, you put yourself in a mental state where you expect others to read and possibly participate from the get go.
+In a way, you change how you perceive your own work.
+
+It's a normal reaction to think that we can do better before getting feedback, which is why you may be tempted to wait until "it's good enough to share".
+But doing so may lead you to wait until very late in the creation process and you may actually never end up opening up for a variety of reasons.
+
+For example, you may have to go back and review the document to ensure you haven't unconsciously added sensitive or classified information, which adds work and delays.
+Or, you may feel less receptive to feedback as you get closer to a final version of a document since you spent so much time aligning your ideas together and wordsmithing.
+Maybe you are worried that getting feedback on a document that is barely an outline will slow you down.
```

Comment:
I would take this sentence out here as we mention it further down and already have two good examples in this paragraph.